doc/docker.md

```diff
@@ -25,11 +25,6 @@ The PostgreSQL database is exposed on localhost port `5432`
 
 `$ psql -h 0.0.0.0 -p 5432 -U postgres -d cexplorer` (then enter secret password)
 
-### To connect to another network:
-```
-NETWORK=testnet docker-compose up && docker-compose logs -f
```

Comment:
instead, once #1253 is merge, we can replace `testnet` with `preprod` (or `preview`).

Reply:
Sure :+1:  @jbgi